### PR TITLE
houdini: maintainer update

### PR DESCRIPTION
**Pull request by permahorse**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by marcusramberg** [comment]
The commit messages and pr title should follow nixpkgs contribution guidelines.

**Review by Aleksanaa — DISMISSED** [review]

File: pkgs/applications/misc/houdini/default.nix
Comment (Aleksanaa) [inline pkgs/applications/misc/houdini/default.nix:104]: Better to avoid `with lib`

**Review by Aleksanaa — COMMENTED** [review]
Please split into two commits:
maintainers: add pedohorse
houdini: move meta to package definition

**Comment by Aleksanaa** [comment]
Would be better to separately package houdini and houdini-unwrapped in pkgs/by-name, though

File: pkgs/applications/misc/houdini/default.nix
Comment (pedohorse) [reply pkgs/applications/misc/houdini/default.nix]: avoided

**Comment by pedohorse** [comment]
> Please split into two commits: maintainers: add pedohorse houdini: move meta to package definition

it's already in 2 commits, but i've just adjusted commit messages

**Comment by pedohorse** [comment]
> The commit messages and pr title should follow nixpkgs contribution guidelines.

sorry for my inattentiveness. 
still not entirely sure how to properly name PRs that are not just version update, is it acceptable now?

**Comment by Aleksanaa** [comment]
> still not entirely sure how to properly name PRs that are not just version update, is it acceptable now?

PR title is not that important, just have to look okay

**Comment by pedohorse** [comment]
> Would be better to separately package houdini and houdini-unwrapped in pkgs/by-name, though

I'm not sure if that's useful, as houdini-unwrapped is only ever needed by the main wrapping package.
but anyway it sounds like a topic for another PR

**Comment by Aleksanaa** [comment]
Yeah, you can do it, or leave it to another one. Just adding another derivation is better for overriding

**Pull request merged**
